src/Sulu/Bundle/MediaBundle/Resources/js/containers/ImageMap/ImageRenderer.js

```diff
@@ -49,7 +50,7 @@ class ImageRenderer extends React.Component<Props> {
             return undefined;
         }
 
-        return '/admin/media/redirect/media/' + imageId + '?locale=' + locale.get();
+        return symfonyRouting.generate('sulu_media.redirect', {id: imageId}) + '?locale=' + locale.get();
```

Comment:
the `locale` can also be given as parameter see examples in: https://github.com/FriendsOfSymfony/FOSJsRoutingBundle/blob/master/Resources/doc/usage.rst

```suggestion
        return symfonyRouting.generate('sulu_media.redirect', {id: imageId, locale: locale.get()});
```